packages/jasmine-auto-spies/src/tests/observable-spy-utils.spec.ts

```diff
@@ -596,6 +596,74 @@ describe('createSpyFromClass - Observables', () => {
       });
     });
 
+    describe(`GIVEN class spy is configured with a manual getter observable property
```

Comment:
I duplicated the tests for the getter, I didn't know what else to do to ensure that a getter will behave as a plain observable property

Reply:
I don't think there's a need to duplicate all of the tests.

Just one test that proves that this bug is fixed should be suffice. 

Can you please remove the rest of the tests and give the `describe` a more specific name to this issue? 

Thanks!